<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWidget</class>
 <widget class="QWidget" name="mainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>959</width>
    <height>518</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="toCubeBtn">
       <property name="text">
        <string>Cube - Куб</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="toTetraBtn">
       <property name="text">
        <string>Tetra - Тетраэдр</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="toCircleBtn">
       <property name="text">
        <string>Circle - Круг</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="toCylinderBtn">
       <property name="text">
        <string>Cylinder - Цилиндр</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="frustumBtn">
       <property name="text">
        <string>Frustum (Перспективная проекция)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="orthoBtn">
       <property name="text">
        <string>Ortho (Ортогональная проекция)</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="glCubeView" name="openGLWidget">
     <property name="minimumSize">
      <size>
       <width>500</width>
       <height>500</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>glCubeView</class>
   <extends>QOpenGLWidget</extends>
   <header>glCubeView.h</header>
   <slots>
    <slot>changeState()</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
